src/NzbDrone.Core/Indexers/RssParser.cs

```diff
@@ -259,9 +259,9 @@ protected virtual RssEnclosure[] GetEnclosures(XElement item)
                                      {
                                          return new RssEnclosure
                                                 {
-                                                    Url = v.Attribute("url").Value,
-                                                    Type = v.Attribute("type").Value,
-                                                    Length = (long)v.Attribute("length")
+                                                    Url = v.Attribute("url")?.Value,
+                                                    Type = v.Attribute("type")?.Value,
+                                                    Length = v.Attribute("length")?.Value?.ParseInt64() ?? default
```

Comment:
`error CS8107: Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.`

At this time we're sticking with C# 7, changing to `0` will resolve.

Reply:
```suggestion
                                                    Length = v.Attribute("length")?.Value?.ParseInt64() ?? 0
```